crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -792,43 +803,145 @@ enum StakeStatus {
 	UNSTAKED
 }
 
+"""
+Parameters related to subsiding staking rewards
+"""
 type StakeSubsidy {
+	"""
+	SUI set aside for stake subsidies -- reduces over time as stake subsidies are paid out over
+	time.
+	"""
 	balance: BigInt
+	"""
+	Number of times stake subsidies have been distributed.
```

Comment:
suggestion: could also mention how often this distribution occurs? iirc it's done per epoch